### PR TITLE
Fixed - IDENTITY-4240

### PR DESCRIPTION
**Pull request by JKAUSHALYA**
https://wso2.org/jira/browse/IDENTITY-4240

### CODE REVIEW

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/SQLQueries.java [hunk 1]
```diff
@@ -63,6 +63,9 @@ private SQLQueries(){
 
         public static final String REMOVE_APPLICATION = "DELETE FROM IDN_OAUTH_CONSUMER_APPS WHERE CONSUMER_KEY=?";
 
+        public static final String UPDATE_OAUTH_INFO = "UPDATE IDN_OAUTH_CONSUMER_APPS SET APP_NAME=? WHERE " +
+                "CONSUMER_KEY=?";
+
         private OAuthAppDAOSQLQueries(){
```
Comment (omindu) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/SQLQueries.java]: Formatting issue.

Comment (JKAUSHALYA) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/SQLQueries.java]: Fixed in https://github.com/wso2/carbon-identity/pull/1690

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java [hunk 2]
```diff
@@ -359,6 +360,36 @@ public void removeConsumerApplication(String consumerKey) throws IdentityOAuthAd
...
+
+            connection = IdentityDatabaseUtil.getDBConnection();
+            statement = connection.prepareStatement(SQLQueries.OAuthAppDAOSQLQueries.UPDATE_OAUTH_INFO);
+
+            statement.setString(1, appName);
+            statement.setString(2, consumerKey);
+
+            statement.execute();
+            connection.commit();
+
```
Comment (mohanvive) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java]: It is better to set the autoCommit false and then do the manual commit..

Comment (mohanvive) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java]: Fix unnecessary line spaces..

Comment (JKAUSHALYA) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java]: Fixed in https://github.com/wso2/carbon-identity/pull/1701

**Pull request merged**